modules/slc/Animal/lib/My/List/Util.pm

```diff
@@ -0,0 +1,18 @@
+package My::List::Util;
+use strict;
+use warnings;
+
+use List::Util qw( );
+$| = 1;
+
+sub sum {
+    my ( $package, @values ) = @_;
+    return List::Util::sum0( @values );
```

Comment:
Hah! So your library just wraps functions from other libraries... this isn't Java, you know... :wink: